### DISCUSSION TITLE
Is it possible to change height of ComboBox?

### DISCUSSION THREAD

**Discussion by danijelVR** [Krypton-Suite/Standard-Toolkit#1935] [Q&A]
Hello,
I am interested in knowing whether is possible to change height of ComboBox because the initial item is not completely visible.
I have tried by changing Size->Height property, but it reverts automatically to 20.
![test1](https://github.com/user-attachments/assets/1f42bd16-7553-4fd0-8bf6-820a78e916aa)
![test2](https://github.com/user-attachments/assets/a75047ec-8e88-4f50-b764-37d2a255b344)

Best,
Danijel

**Comment by Smurf-IV** [comment]
Can you state what version of the Toolkit you are using please ?

**Reply by MattH-Work** [reply]
you can alter the height by setting the minimum & maximum size values

![image](https://github.com/user-attachments/assets/4b40b7ea-e80c-4f01-99a3-59509a297d1a)

**Comment by giduac** [comment]
Hi @danijelVR,

Have a look at #1832, this will fix this.

**Comment by danijelVR** [comment]
Hello guys,
I really appologize for my late response. 
Because of the private issues and did not work on my projects for a while. I just want to thank you for all your responses. 
Regarding the combobox issues, I found something else more but need to check first. Maybe, it is already fixed in the latest Toolkit verson. 😉

Best,
Danijel
